fn_ipvoid/README.md

```diff
@@ -0,0 +1,35 @@
+## About IPVOID
```

Comment:
Update this `README.md` file to follow this template: https://github.com/ibmresilient/resilient-community-apps/blob/master/fn_digital_shadows_search/README.md

Ensure we document the function inputs and output